src/chrome/content/rules/Google.xml

```diff
@@ -4,437 +4,113 @@
+	<test url="http://google.co.ug/" />
+	<test url="http://www.google.co.ug/" />
+	<test url="http://google.co.uk/" />
+	<test url="http://www.google.co.uk/" />
+	<test url="http://google.co.za/" />
+	<test url="http://www.google.co.za/" />
+
+	<test url="http://google.co/" />
+	<test url="http://www.google.co/" />
+	<test url="http://google.fr/" />
+	<test url="http://www.google.fr/" />
+	<test url="http://google.de/" />
+	<test url="http://www.google.de/" />
+	<test url="http://google.lk/" />
+	<test url="http://www.google.lk/" />
+	<test url="http://google.in/" />
+	<test url="http://www.google.in/" />
+	<test url="http://google.kr/" />
+	<test url="http://www.google.kr/" />
+	<test url="http://google.nl/" />
+	<test url="http://www.google.nl/" />
+	<test url="http://google.ru/" />
+	<test url="http://www.google.ru/" />
+	<test url="http://google.sk/" />
+	<test url="http://www.google.sk/" />
+	<test url="http://google.ua/" />
+	<test url="http://www.google.ua/" />
+
+	<test url="http://google.com.au/" />
+	<test url="http://www.google.com.au/" />
+	<test url="http://google.com.bo/" />
+	<test url="http://www.google.com.bo/" />
+	<test url="http://google.com.co/" />
+	<test url="http://www.google.com.co/" />
+	<test url="http://google.com.ec/" />
+	<test url="http://www.google.com.ec/" />
+	<test url="http://google.com.hk/" />
+	<test url="http://www.google.com.hk/" />
+	<test url="http://google.com.kz/" />
+	<test url="http://www.google.com.kz/" />
+	<test url="http://google.com.mx/" />
+	<test url="http://www.google.com.mx/" />
+	<test url="http://google.com.ru/" />
+	<test url="http://www.google.com.ru/" />
+	<test url="http://google.com.sg/" />
+	<test url="http://www.google.com.sg/" />
+	<test url="http://google.com.sl/" />
+	<test url="http://www.google.com.sl/" />
+	<test url="http://google.com.vn/" />
+	<test url="http://www.google.com.vn/" />
+
+	<test url="http://www.google.ca/about" />
+	<test url="http://www.google.com/about" />
+	<test url="http://www.google.com.au/about" />
+	<test url="http://www.google.ca/accounts" />
+	<test url="http://www.google.com/accounts" />
+	<test url="http://www.google.com.au/accounts" />
+	<test url="http://www.google.ca/admob" />
+	<test url="http://www.google.com/admob" />
+	<test url="http://www.google.ca/adplanner" />
+	<test url="http://www.google.com/adplanner" />
+	<test url="http://www.google.com.au/adplanner" />
+	<test url="http://www.google.ca/ads" />
+	<test url="http://www.google.com/ads" />
+	<test url="http://www.google.com.au/ads" />
+
+	<test url="http://google.com/search?q=test&amp;tbm=isch" />
+	<test url="http://www.google.com/search?q=test&amp;tbm=isch" />
+	<test url="http://www.google.com/search?q=foobar&amp;tbm=isch" />
+	<test url="http://www.google.com/search?q=foobar&amp;tbm=isch&amp;something=baz" />
+
+
+	<!-- https://github.com/EFForg/https-everywhere/issues/2747 -->
+	<!--<exclusion pattern="^http://(www\.)?google\.cn" />-->
+
+
+	<securecookie host=".+" name=".+" />
```

Comment:
Since subdomains are covered in different rulesets I would limit this to the TLD and www.
But I wasn't able to find any issues and it would be so important to have such a cookie rule for all Google services. I hope @gloomy-ghost could also give a feedback on this new change.

Reply:
If I understand correctly, `securecookie` only applies to targets that are in the same ruleset, so in this case it is limited to the TLD and www